fuseops/ops.go

```diff
@@ -500,6 +500,14 @@ type OpenDirOp struct {
 	// a later call to ReleaseDirHandle.
 	Handle    HandleID
 	OpContext OpContext
+
+	// CacheDirContentAsPageCache conveys the kernel to cache the response of next
+	// ReadDirOp as page cache. Once cached, listing on that directory will be
+	// served from the kernel until invalidated.
+	CacheDirContentAsPageCache bool
+
+	// KeepDirContentPageCache used to invalidate the cached directory content.
+	KeepDirContentPageCache bool
```

Comment:
Please rename to `KeepCache` for consistency with the Linux kernel flag name (`FOPEN_KEEP_CACHE`). Also, the kernel documents this field as “don't invalidate the data cache on open”, so I think a better description is “KeepCache instructs the kernel to not invalidate the data cache on open calls”